### PR TITLE
Add details of configuring third party services for local development.

### PR DESCRIPTION
**Pull request by ntoll**
## What

More work on documentation, a follow-up to #803.

## How to review

Follow the steps from a fresh install and check you have a working dev environment..!

### CODE REVIEW

**Comment by xoen** [comment]
FYI: @ntoll docker build in CircleCI seems to fail, it sounds like the `ca-certificates` problem I solved, rebase `master` may fix that for you as well.

File: doc/running.md [hunk 1]
```diff
@@ -201,6 +160,150 @@ DJANGO_SETTINGS_MODULE=controlpanel.settings.test pytest
...
+
+This isn't strictly required, so feel free to skip this step.
+
+```sh
+python3 manage.py createsuperuser
+```
+
+Your `Username` needs to be your GitHub username.
+Your `Auth0 id` needs to be the number associated with you in auth0.com and
+labelled `user_id` (not working for me yet).
```
Comment (xoen) [inline doc/running.md:306]: maybe tweak to:

> Your `Auth0 id` needs to be the number associated with you in auth0.com and
labelled `user_id` (including the `github|` prefix, e.g. `github|123...`).

If I understand correctly, this is now working fine for you now right?

File: doc/running.md [hunk 2]
```diff
@@ -215,4 +318,5 @@ Or with Gunicorn WSGI server:
 ```sh
 gunicorn -b 0.0.0.0:8000 -k uvicorn.workers.UvicornWorker -w 4 controlpanel.asgi:application
 ```
-Go to http://localhost:8000/
+Go to http://localhost:8000/, sign in via Auth0 and marvel at your locally
+runing control panel.
```
Comment (xoen) [inline doc/running.md:322]: 😍

Comment (ntoll) [reply doc/running.md:322]: :slightly_smiling_face:

File: doc/running.md (hunk 1 above)
Comment (ntoll) [reply doc/running.md:306]: Nope. I don't honestly think this step is required.

**Pull request merged**